docs/plugins/devtools.md

```diff
@@ -39,7 +39,11 @@ The plugin supports the following options passed via the `forRoot` method:
 - `name`: Set the name by which this store instance is referenced in devtools (Default: 'NGXS')
 - `disabled`: Disable the devtools during production
 - `maxAge`: Max number of entries to keep.
-- `actionSanitizer`: Reformat actions before sending to dev tools
+- `maxAge`: Max number of entries to keep.
```

Comment:
Looks like `maxAge` has been repeated and we have lost the description for `actionSanitizer`.

Reply:
@markwhitfeld yes, fixed